lib/bugsnag/loggers/logger.rb

```diff
@@ -0,0 +1,68 @@
+require "logger"
+require "bugsnag"
+
+module Bugsnag::Loggers
```

Comment:
Is there any benefit to the Loggers module? `Bugsnag::Logger` reads nicer. Is there a clash with our internal logger maybe?

Reply:
There's going to be some shared functionality between the `Logger` and the `Appender` classes (Appender coming soon).  I've thought about renaming the module 'logging' as it's what the module deals with, but it will contain 'loggers', so not sure about that